Makefile

```diff
@@ -3,17 +3,20 @@
 PROJECT = probfit
 CYTHON ?= cython
 
+PYX_FILES := $(wildcard probfit/*.pyx)
+
+.PHONY: help clean build test coverage doc doc-show code-analysis flake8 pylint
+
 help:
 	@echo ''
 	@echo ' probfit available make targets:'
 	@echo ''
 	@echo '     help             Print this help message (the default)'
 	@echo ''
 	@echo '     clean            Remove generated files'
-	@echo '     build            Build inplace'
+	@echo '     build            Build Cython files'
```

Comment:
I value consistency. Under the iminuit banner, I think it's nice to have the two Makefiles behave in a similar way. I'll change to the previous wording.